examples/bigcommerce/lib/bigcommerce/api.ts

```diff
@@ -17,12 +16,12 @@ const A_WEEK_FROM_NOW = Math.floor(
 export async function getApiToken(): Promise<string> {
```

Comment:
Is `getApiToken` still needed @joshuawootonn? Maybe we can remove it if we don't use it anymore.

Reply:
Yeah don't need it! I will remove it.

FWIW this is the reason I didn't use their env before. But I will update the readme to have a curl request for getting the storefront token in the case of them not using the integration